lib/src/hit_area_controls.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:flutter/material.dart';
+
+import 'center_play_button.dart';
+import 'seek_rewind_button.dart';
+
+class HitAreaControls extends StatelessWidget {
+  const HitAreaControls({
+    Key? key,
+    this.onTapPlay,
+    this.onPressedPlay,
+    this.seekRewind,
+    this.seekForward,
+    required this.backgroundColor,
+    required this.iconColor,
+    required this.isFinished,
+    required this.isPlaying,
+    required this.showPlayButton,
+    required this.showSeekButton,
+  }) : super(key: key);
+
+  final Function()? onTapPlay;
+  final Function()? onPressedPlay;
+  final Function()? seekRewind;
+  final Function()? seekForward;
+  final Color backgroundColor;
+  final Color iconColor;
+  final bool isFinished;
+  final bool isPlaying;
+  final bool showPlayButton;
+  final bool showSeekButton;
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.center,
+      children: [
+        _buildSeekRewindButton(isSeekForward: false),
+        GestureDetector(
+          onTap: onTapPlay,
+          child: CenterPlayButton(
+            backgroundColor: backgroundColor,
+            iconColor: iconColor,
+            isFinished: isFinished,
+            isPlaying: isPlaying,
+            show: showPlayButton,
+            onPressed: onPressedPlay,
+          ),
+        ),
+        _buildSeekRewindButton(isSeekForward: true),
+      ],
+    );
+  }
+
+  SeekRewindButton _buildSeekRewindButton({bool isSeekForward = true}) {
+    return SeekRewindButton(
+      backgroundColor: backgroundColor,
+      iconColor: iconColor,
+      show: showSeekButton,
+      onPressed: isSeekForward ? seekForward : seekRewind,
+      onDoublePressed: isSeekForward ? seekForward : seekRewind,
+      icon: isSeekForward ? Icons.fast_forward : Icons.fast_rewind,
+    );
+  }
```

Comment:
Do not use methods to return widgets. Instead you can simply add `SeekRewindButton` as part of your `Row` children.

Reply:
earlier it was inside a Row children. it's extracted into method as changes requested https://github.com/fluttercommunity/chewie/pull/717#discussion_r1250836643

Reply:
@diegotori  `SeekRewindButton `  is already extracted as Stateless Widget lib/src/seek_rewind_button.dart

Reply:
@punit1111 that is true, but instead of creating it from `_buildSeekRewindButton`, you should just replace those calls with the actual widget:

Change this:
```dart
_buildSeekRewindButton(isSeekForward: true),
```

to this:

```dart
SeekRewindButton(
      backgroundColor: backgroundColor,
      iconColor: iconColor,
      show: showSeekButton,
      onPressed: isSeekForward ? seekForward : seekRewind,
      onDoublePressed: isSeekForward ? seekForward : seekRewind,
      icon: Icons.fast_forward,
    )
```

Reply:
Do likewise for the other places where that method is being called from.